packages/editor-ui/src/event-bus/global-link-actions.ts

```diff
@@ -0,0 +1,16 @@
+import { createEventBus } from 'n8n-design-system/utils';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type LinkActionFn = (...args: any[]) => void;
+
+export type RegisterCustomActionOpts = {
+	key: string;
+	action: LinkActionFn;
+};
+
+export interface GlobalLinkActionsEventBusEvents {
+	/** See useGlobalLinkActions.ts */
+	registerCustomAction: RegisterCustomActionOpts;
+}
+
+export const globalLinkActionsEventBus = createEventBus();
```

Comment:
Shouldn't this be:

```suggestion
export const globalLinkActionsEventBus = createEventBus<GlobalLinkActionsEventBusEvents>();
```

?

Reply:
Oh yes, yes it should 🙈